test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -808,6 +811,1175 @@ public void ProjectReferenceBetweenTwoGeneratingProjects(bool internalsVisibleTo
         this.AssertNoDiagnostics();
     }
 
+    [Fact]
```

Comment:
How should I think about the tests your adding here, which use verbatim text assertions compared to my other tests that mostly assert specific things? I can imagine a *few* verbatim tests, but each new one creates a maintenance burden when either the generator or the metadata behind it changes. I see you added *several*.

Reply:
One option is to extract the output **BOOL.g.cs** to a shared constant, so if it changes you can change all cases at once.

Another option is to generate one or more outputs by running the generator with `<EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>` ([example](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/a559b3e58af09387aadc1372a504f9208bce4909/StyleCop.Analyzers/StyleCop.Analyzers/StyleCop.Analyzers.csproj#L26-L29)), and then reading files from that output location during tests. Changes would automatically apply to all tests using that output, and the diffs would be visible in source control.